### PR TITLE
swig: add page

### PR DESCRIPTION
**Pull request by alufers**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/swig.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# swig
+
+> Generate bindings between C/C++ code and various high level languages (Go, Java, Javascript, Perl, PHP, Python, Tcl, Ruby, C#, D).
```
Comment (zdroid) [inline pages/common/swig.md]: This and the next line need to be rewritten. They contain too much information for someone who just wants to get a quick info about this program.

The rest is all well done. :+1:

**Comment by zdroid** [comment]
I edited your file to fix the Travis errors. Sentences in description should end in periods, and there should be a newline after the description is done.

File: pages/common/swig.md (hunk 1 above)
Comment (sbrl) [reply pages/common/swig.md]: Again @zdroid, offering a solution is probably a good idea. For example:
```suggestion
> Generate bindings between C / C++ code and various high level languages such as Javascript, Python, C#, and more.
```

**Comment by schneiderl** [comment]
I'm sorry I didn't mean to merge this already. If someone finds anything to be changed please let me know that I'll revert the merge.

**Pull request merged**